src/components/map/Mapbox.vue

```diff
@@ -55,6 +59,16 @@ export default {
     onMapLoaded(event) {
       // console.log("Map loaded!");
       this.map = event.map;
+    },
+    navigateToEvent(event) {
+      const { id } = event.priv;
+      this.$router.push({
+        name: "event-page",
+        params: {
```

Comment:
Just a note that when we're using the API, we can't pass params but need to use what's in the url to make API calls and retrieve the appropriate data

Reply:
ya